pkgs/applications/networking/ndppd/default.nix

```
@@ -1,24 +1,21 @@
-{ lib, stdenv, fetchFromGitHub, gzip, nixosTests }:
+{ lib, stdenv, fetchurl, nixosTests }:
 
 stdenv.mkDerivation rec {
   pname = "ndppd";
-  version = "0.2.5";
+  version = "0.2.5.43";
```

```suggestion
  version = "0.2.5-unstable-2023-12-11";
```
This is the nixpkgs convention for such versions.

---

SUSE definition of 0.2.5.43 means commit eb81b8f, 43 commits after 0.2.5 (committed on 2017-09-20).
https://github.com/DanielAdolfsson/ndppd/compare/0.2.5...eb81b8f

---

Has the 0.2.5-unstable-2023-12-11 you mentioned been verified? Can it be considered officially released?

---

I suppose not. In that case you can use 2017-09-20

---

So you look at the version number 0.2.5-unstable-2017-09-20 and you can realize that this is the same version as SUSE's 0.2.5.43?
https://repology.org/project/ndppd/versions
https://pkgs.org/search/?q=ndppd